server/apps/immich/src/api-v1/asset/asset.controller.ts

```diff
@@ -251,6 +251,21 @@ export class AssetController {
     return this.assetService.getAllAssets(authUser, dto);
   }
 
+  /**
+   * Get all AssetEntity by userId
+   */
+  @Authenticated()
+  @Get('/partner/:id')
+  @ApiHeader({
+    name: 'if-none-match',
+    description: 'ETag of data already cached on the client',
+    required: false,
+    schema: { type: 'string' },
+  })
+  getUserAssets(@GetAuthUser() authUser: AuthUserDto, @Param('id') id: string): Promise<AssetResponseDto[]> {
```

Comment:
This endpoint can be called with any user ID and it will return their assets, because there's no partner validation. Also the `id` param could use better validation: `@Param() { id }: UUIDParamDto`

Reply:
Removed `getUserAssets` method. I forgot to delete it while I was moving from `GalleryViewer` to `AssetGrid`